services/transaction/tests/schemas/transactionSchema.schema.js

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I think this change would be unnecessary because in transaction service we do not have any implementation related to `version 1`.Also, if we want to add this change, we need to do it in other services (newsfeed and market) as well for consistency.

Reply:
Making the explicit separation is good to easily not confuse the schemas corresponding to the `v1` and `v2` APIs.
Yes, you are right, we need to make the changes gradually to achieve consistency. Housekeeping duties await us 😄